extensions/wikia/AdEngine/js/provider/gpt/helper.js

```diff
@@ -32,7 +32,6 @@ define('ext.wikia.adEngine.provider.gpt.helper', [
 	'use strict';
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Use of JavaScript strict mode may result in unexpected behaviour in some browsers. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AStrictMode)